src/main/java/com/iota/iri/storage/rocksDB/RocksDBPersistenceProvider.java

```diff
@@ -436,52 +478,29 @@ private void initDB(String path, String logPath, Map<String, Class<? extends Per
                 throw e;
             }
 
-            File pathToLogDir = Paths.get(logPath).toFile();
-            if (!pathToLogDir.exists() || !pathToLogDir.isDirectory()) {
-                boolean success = pathToLogDir.mkdir();
-                if (!success) {
-                    log.warn("Unable to make directory: {}", pathToLogDir);
-                }
-            }
-
-            int numThreads = Math.max(1, Runtime.getRuntime().availableProcessors() / 2);
-            RocksEnv.getDefault()
-                .setBackgroundThreads(numThreads, RocksEnv.FLUSH_POOL)
-                .setBackgroundThreads(numThreads, RocksEnv.COMPACTION_POOL);
-
-            options = new DBOptions()
-                .setCreateIfMissing(true)
-                .setCreateMissingColumnFamilies(true)
-                .setDbLogDir(logPath)
-                .setMaxLogFileSize(SizeUnit.MB)
-                .setMaxManifestFileSize(SizeUnit.MB)
-                .setMaxOpenFiles(10000)
-                .setMaxBackgroundCompactions(1);
-
-            options.setMaxSubcompactions(Runtime.getRuntime().availableProcessors());
+            options = createOptions(logPath, configFile);
 
             bloomFilter = new BloomFilter(BLOOM_FILTER_BITS_PER_KEY);
-
-            BlockBasedTableConfig blockBasedTableConfig = new BlockBasedTableConfig().setFilter(bloomFilter);
+            cache = new LRUCache(cacheSize * SizeUnit.KB, 2);
+            compressedCache = new LRUCache(32 * SizeUnit.KB, 10);
```

Comment:
Those changes resulted in performance benefits? Or was it something you had to do due to RocksDb upgrade